portability-worker/src/main/java/org/dataportabilityproject/worker/WorkerImpl.java

```diff
@@ -16,19 +16,25 @@
 package org.dataportabilityproject.worker;
 
 import com.google.common.base.Preconditions;
+import com.google.gson.Gson;
 import com.google.inject.Inject;
 import java.io.IOException;
 import org.dataportabilityproject.PortabilityCopier;
 import org.dataportabilityproject.ServiceProviderRegistry;
 import org.dataportabilityproject.cloud.interfaces.CloudFactory;
+import org.dataportabilityproject.job.Crypter;
+import org.dataportabilityproject.job.CrypterFactory;
 import org.dataportabilityproject.job.JobDao;
 import org.dataportabilityproject.job.PortabilityJob;
 import org.dataportabilityproject.shared.PortableDataType;
+import org.dataportabilityproject.shared.auth.AuthData;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 final class WorkerImpl {
-  private final Logger logger = LoggerFactory.getLogger(WorkerImpl.class);
+  private static final Logger logger = LoggerFactory.getLogger(WorkerImpl.class);
+  private static final Gson GSON = new Gson();
```

Comment:
Do you think we should switch to Jackson since we are using that elsewhere?

Reply:
I added a TODO for now to use the Jackson-based AuthData when it's checked in.